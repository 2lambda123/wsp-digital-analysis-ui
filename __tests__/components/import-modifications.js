/* global describe, it, expect, jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { mockComponents } from '../../testUtils'

jest.mock('react-select', () => 'React-Select')
jest.mock('../../lib/components/buttons', () => { return mockComponents(['Button']) })
jest.mock('../../lib/components/panel', () => { return mockComponents(['Panel', 'Heading', 'Body']) })

import ImportModifications from '../../lib/components/import-modifications'

describe('Component > ImportModifications', () => {
  it('renders correctly', () => {
    const copyFromScenarioFn = jest.fn()
    const tree = renderer.create(
      <ImportModifications
        copyFromScenario={copyFromScenarioFn}
        toScenarioId='1'
        variants={[]}
        />
    ).toJSON()
    expect(tree).toMatchSnapshot()
    expect(copyFromScenarioFn).not.toBeCalled()
  })
})
