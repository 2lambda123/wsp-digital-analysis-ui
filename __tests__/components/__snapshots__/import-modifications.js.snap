exports[`Component > ImportModifications renders correctly 1`] = `
<Panel>
  <Heading>
    Import modifications from another scenario
  </Heading>
  <Body>
    <div
      className="form-group">
      <React-Select
        onChange={[Function]}
        options={undefined}
        placeholder="Select a scenario..."
        value={undefined} />
    </div>
    <Button
      block={true}
      disabled={true}
      onClick={[Function]}
      style="success">
      Import modifications
    </Button>
  </Body>
</Panel>
`;
